tools/find_dali.cmake

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Wondering, why isn't this file in cmake/ dir?

Reply:
The answer is here: https://github.com/NVIDIA/DALI/pull/2325#discussion_r500131929
:)